lago/virt.py

```diff
@@ -166,6 +167,32 @@ def virt_path(self, *args):
     def bootstrap(self):
         utils.invoke_in_parallel(lambda vm: vm.bootstrap(), self._vms.values())
 
+    def export_entities(self, entities_names, standalone, dst_dir, compress):
```

Comment:
Missing docstrings

something not critical - but I think 'entities' as  a name doesn't mean anything, it would be hard to deduce from the name only what this function does.